pandas/tests/series/test_dtypes.py

```diff
@@ -487,3 +487,25 @@ def test_reindex_astype_order_consistency(self):
         s1 = s.reindex(new_index).astype(temp_dtype).astype(new_dtype)
         s2 = s.astype(temp_dtype).reindex(new_index).astype(new_dtype)
         tm.assert_series_equal(s1, s2)
+
+    @pytest.mark.parametrize(
+        "stup",
```

Comment:
You can do the tuple unpacking here. This can be `"series, dtype"` and the function can take those two arguments.

Reply:
fixed